ci/scripts/build.sh

```diff
@@ -59,3 +63,11 @@ buildkite-agent artifact upload risedev-dev-"$profile"
 buildkite-agent artifact upload risingwave_regress_test-"$profile"
 buildkite-agent artifact upload ./sqlsmith-"$profile"
 buildkite-agent artifact upload ./compaction-test-"$profile"
+
+echo "--- Build Java connector node"
```

Comment:
Can we download the artifact directly, or moving this build to a separate phase?

Reply:
Besides, there seems too many logs generated by maven. Can we resolve this?

Reply:
I think this script is for building. And e2e test of remote sinks also depends on this component.

Reply:
Since it has nothing to do with Rust components, I think we can paralyze them?

Reply:
We can separate it into another job, e.g. `build-other` then we can parallel it with the `build` job. cc @huangjw806